src/feature/MeetTheHost.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import "./MeetTheHost.css";
+
+const MeetTheHost = () => {
+  return (
+    <div className="host-info-container">
+      <h2 className="header">Meet your Host</h2>
+      <div className="main-container">
+        <div className="leaf-side">
+          <div className="round-photo-component">
+            <img
+              className="raus-img"
+              src="https://a0.muscache.com/im/pictures/user/d62627ea-ea22-4cf1-b38a-152f1f86a9ed.jpg"
+              alt="RAUS"
+            />
+          </div>
+          <div className="verified-container">
+            <svg
+              version="1.1"
+              xmlns="http://www.w3.org/2000/svg"
+              width="32"
+              height="32"
+              viewBox="0 0 32 32"
+              role="img"
+              aria-label="check mark"
+              className="verifiedicon-svg"
+            >
+              <title>verified-sheild</title>
+              <path d="M13.876 21.464l8.732-8.732-1.532-1.532-7.2 7.149-2.809-2.809-1.532 1.532 4.341 4.392zM16.071 4l9.804 4.392v6.536c0 6.026-4.187 11.694-9.804 13.072-5.617-1.379-9.804-7.047-9.804-13.072v-6.536l9.804-4.391z"></path>
+            </svg>
+          </div>
+          <div className="raus-superhost-container">
+            <h2 className="raus-info">Raus</h2>
```

Comment:
It seems to me that "Raus" is a name of a concrete host. We should avoid including such data into things like css class names because eventually this component will be used to display any user or host.

Reply:
Thank you so much for this comment. I always have a problem with naming CSS classes :) I will change the names.